app/javascript/gobierto_plans/webapp/components/Project.vue

```diff
@@ -53,7 +53,8 @@ export default {
       const { uid } = item.attributes;
       const value = attributes[uid];
 
-      if (show_empty_fields || (value && value.length)) {
+      // hide "estat" custom_field on purpose (it's a vocabulary also)
+      if ((uid !== "estat") && (show_empty_fields || (value && value.length)) ){
```

Comment:
i18n issues here?

Reply:
That's the _id_ (or _slug_) of the custom field, unlike their translations. So it won't be problematic as long as the id matches. This is the reason I did mention about the use of `options.configuration` object, but that requires backend stuff.